math/median.go

```diff
@@ -0,0 +1,32 @@
+// author(s) [jo3zeph](https://github.com/jo3zeph)
+// description: Find the median from a set of values
+// see median_test.go
+
+package math
+
+import (
+	"github.com/TheAlgorithms/Go/constraints"
+	"github.com/TheAlgorithms/Go/sort"
+)
+
+func Median[T constraints.Number](values []T) float64 {
+
+	sort.Bubble(values)
+
+	var median float64
+
+	l := len(values)
+
+	switch {
+	case l == 0:
+		return 0
+
+	case l%2 == 0:
+		median = float64((values[l/2-1] + values[l/2]) / 2)
+
+	default:
+		median = float64(values[l/2])
+	}
+
+	return median
```

Comment:
This would be redundant after those returns and you can remove the declaration of median on line 16 as well.